packages/utils/src/index.js

```diff
@@ -36,13 +36,20 @@ if (__DEV__) {
 
 export { checkStyles };
 
-export let wrapEvent = (theirHandler, ourHandler) => event => {
+export const wrapEvent = (theirHandler, ourHandler) => event => {
   theirHandler && theirHandler(event);
   if (!event.defaultPrevented) {
     return ourHandler(event);
   }
 };
 
+export const callEventWithDefault = (theirHandler, ourHandler) => event => {
```

Comment:
It looks like you're not actually using this anywhere in this PR. I think we can leave it out?